.ci/docker/Dockerfile_dist_s1

```diff
@@ -0,0 +1,70 @@
+# Dockerfile to produce the production DSWx-NI PGE Docker image for OPERA
+# Authors: Scott Collins, Ray Bambery
+
+# Default SAS image path, must be provided by the docker build call via --build-arg
+ARG SAS_IMAGE
+FROM $SAS_IMAGE
+
+ARG PGE_SOURCE_DIR
+ARG PGE_DEST_DIR=/home/ops
+ARG CONDA_ROOT=/home/ops/miniconda3
+
+ENV PGE_DEST_DIR=$PGE_DEST_DIR
+ENV CONDA_ROOT=$CONDA_ROOT
+
+ARG BUILD_DATE_TIME
+ARG BUILD_VERSION
+
+# labels
+# the label-schema convention: http://label-schema.org/rc1/
+LABEL org.label-schema.build-date=${BUILD_DATE_TIME} \
+  org.label-schema.version=${BUILD_VERSION} \
+  org.label-schema.license="Copyright 2024,\
+ by the California Institute of Technology.\
+ ALL RIGHTS RESERVED.\
+ United States Government sponsorship acknowledged.\
+ Any commercial use must be negotiated with the Office of Technology Transfer\
+ at the California Institute of Technology.\
+ This software may be subject to U.S. export control laws and regulations.\
+ By accepting this document, the user agrees to comply with all applicable\
+ U.S. export laws and regulations. User has the responsibility to obtain\
+ export licenses, or other export authority as may be required, before\
+ exporting such information to foreign countries or providing access to\
+ foreign persons." \
+  org.label-schema.name="OPERA Product Generation Executable (PGE) Image" \
+  org.label-schema.schema-version="1.0" \
+  org.label-schema.vendor="California Institute of Technology" \
+  maintainer="California Institute of Technology"
+
+# Copy the OPERA PGE software into the container
+# the base container has a default user "mamba" with UID/GID 1000/1000
+COPY --chown=dist_user:dist_user ${PGE_SOURCE_DIR} ${PGE_DEST_DIR}
+
+# Switch to root for installing into Conda Env
+USER 0:0
+
+# Set the default shell to run commands within the Conda environment named "DSXW-SAR" using bash
+SHELL ["conda", "run", "-n", "dist-s1-env", "/bin/bash", "-c"]
+
+# Install dependencies into existing Conda Env
+# Modifications to the destination dir permissions are also required here to ensure
+# we don't run into issues when running as an outside user (via docker run -u)
+RUN set -ex \
+    && cd ${PGE_DEST_DIR} \
+    && mkdir -p ${CONDA_ROOT}/bin \
+    && cp ${PGE_DEST_DIR}/opera/scripts/*_entrypoint.sh ${CONDA_ROOT}/bin \
+    && chmod +x ${CONDA_ROOT}/bin/*_entrypoint.sh \
+    && apt-get update \
+    && apt-get install --no-install-recommends -y g++ \
+    && apt-get clean \
+    && python -m pip install -r ${PGE_DEST_DIR}/opera/requirements.txt \
```

Comment:
The pip install is failing without g++ installed
We should coordinate with ADT to have a numpy image <2 installed in the SAS so we won't have to worry about rebuilding it here, but it seems the issue is also present in the mgrs install.